tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -179,24 +228,9 @@ static void gatt_scan_cb(const bt_addr_le_t *addr, int8_t rssi,
 		parse_mesh_proxy_service(buf);
 	}
 
-	if (cnt) {
-		int64_t delta = k_uptime_delta(&timestamp);
-
-		LOG_INF("rx: %s +%d ms", txt_msg, delta);
-
-		ASSERT_TRUE(delta >= gatt_param.interval &&
-			delta < (gatt_param.interval + 15));
-	} else {
-		timestamp = k_uptime_get();
+	LOG_INF("rx: %s", txt_msg);
 
-		LOG_INF("rx: %s +0 ms", txt_msg);
-	}
-
-	cnt++;
-
-	if (cnt == gatt_param.transmits) {
-		cnt = 0;
-		timestamp = 0;
+	if (check_delta_time(gatt_param.transmits-1, gatt_param.interval)) {
```

Comment:
Could you please add a comment on why `gatt_param.transmits - 1` is used. Now it seems a bit weird since the test initializes the `transmits` value. Why did we initialize it in some value that requires subtraction 1?
It should be clarified adv uses retransmit but gatt uses transmit for the function.